contrib/src/main/java/com/datatorrent/contrib/cassandra/AbstractCassandraTransactionableOutputOperator.java

```diff
@@ -48,29 +54,58 @@
  * @param <T>type of tuple</T>
  * @since 1.0.2
  */
-public abstract class AbstractCassandraTransactionableOutputOperator<T> extends AbstractBatchTransactionableStoreOutputOperator<T, CassandraTransactionalStore> {
+public abstract class AbstractCassandraTransactionableOutputOperator<T> extends AbstractBatchTransactionableStoreOutputOperator<T, CassandraTransactionalStore> implements ActivationListener<Context.OperatorContext>
+{
+  private transient PreparedStatement updateCommand;
 
-  public AbstractCassandraTransactionableOutputOperator(){
-    super();
+  @Override
+  public void activate(OperatorContext context)
+  {
+    updateCommand = getUpdateCommand();
   }
 
   /**
+   * Gets the statement which insert/update the table in the database.
+   *
+   * @return the cql statement to update a tuple in the database.
+   */
+  @Nonnull
+  protected abstract PreparedStatement getUpdateCommand();
+
+  /**
    * Sets the parameter of the insert/update statement with values from the tuple.
    *
    * @param tuple     tuple
    * @return statement The statement to execute
    * @throws DriverException
+   *
+   * @deprecated since 3.5.0.0 use {@link #setStatementParameters(PreparedStatement, T)} instead.<br/>
    */
+  @Deprecated
   protected abstract Statement getUpdateStatement(T tuple) throws DriverException;
```

Comment:
Will removing this cause any issues with backward compatibility. Keeping this (even deprecated) may require us to ensure that calls to ```getUpdateStatement()``` are successful.
@PramodSSImmaneni What is the right way to get rid of this method?